### PR TITLE
Kpod pull

### PR DESCRIPTION
**Pull request by baude**
## Description
pull and delete

## Related Bugzillas
-
-

## Related Issue Numbers
-
-

## Pull Request Checklist:

If your Pull request contains new features or functions, tests are required. If the PR is a bug fix and no tests exist, please consider adding some to prevent regressions.
- [ ] Unittests
- [ ] Integration Tests

### CODE REVIEW

File: Atomic/backends/_containers_storage.py [hunk 1]
```diff
@@ -53,7 +60,26 @@ def pull_image(self, image, remote_image_obj, **kwargs):
...
+            image += ":{}".format(tag)
+        if '@sha256:' in image:
+            image = image.replace("@sha256:", ":")
+
+        insecure = False
+        registries_config = util.load_registries_from_yaml()
+        if "insecure_registries" in registries_config:
+            if registry in registries_config['insecure_registries']:
+                insecure = True
+        source = "docker://{}".format(image)
```
Comment (TomSweeneyRedHat) [inline Atomic/backends/_containers_storage.py]: will the source/dest always be hardcoded?  Can/should we have them as parameters, perhaps optional, to this function?

Comment (baude) [reply Atomic/backends/_containers_storage.py]: Yes they are for now.  We haven't talked about opening up the transports yet.  Each other backend is more or less this way.  Certainly something that we can discuss in the future but the assumption was fancy stuff be done with skopeo or the individual tools

Comment (rhatdan) [reply Atomic/backends/_containers_storage.py]: Could check to see if the transport is specified?
atomic pull --storage container-storage dir:/tmp/alpine.tar 

But I agree we should move this to a future version.

**Comment by rhatdan** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Thanks for the 411.  I'm fine with leaving as is for now, but please create a trello card or issue to make sure we don't lose track of the hardcoded transports.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ 22a56e2

**Comment by baude** [comment]
[pull_image [crio-atomic]](https://trello.com/c/BjOV6Vuk/427-pullimage-crio-atomic)

**Comment by baude** [comment]
[delete_image [crio-atomic]](https://trello.com/c/izv3yGIC/422-deleteimage-crio-atomic)

**Comment by baude** [comment]
[inspect image [crio-atomic]](https://trello.com/c/Kmyede5S/429-inspect-image-crio-atomic)

**Comment by baude** [comment]
@rh-atomic-bot  r+ 0f78a60

**Comment by baude** [comment]
[inspect image [crio-atomic]](https://trello.com/c/Kmyede5S/429-inspect-image-crio-atomic)

**Pull request closed without merging**